public/js/user.js

```diff
@@ -151,9 +151,9 @@ apos.define('apostrophe-i18n-debugger', {
 
 $(function() {
```

Comment:
Oops, this whole block is a completely superfluous pair of inline tests I ran while making sure these events worked. You can remove it instead.

Reply:
If you don't mind. Not your bad.